chocolatey/chocolatey-au.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>chocolatey-au</id>
+    <title>Chocolatey Automatic Package Updater Module</title>
+    <version>$version$</version>
+    <authors>Miodrag Milić, Chocolatey Community</authors>
+    <owners>Miodrag Milić, Chocolatey Community</owners>
+    <summary>Chocolatey-AU is Powershell module that helps you to automate Chocolatey package updates</summary>
+    <description></description>

```

Comment:
Is this empty on purpose? Or did I somehow miss it being set somewhere else. A description would be something I would consider to be required.

Reply:
It takes the Summary and the `Features` section of the Readme and sets the description during the build:

```powershell
    $description = $chocolateyPackage.package.metadata.summary + ".`n`n" + $features
    $chocolateyPackage.package.metadata.description = $description
```